upstream-sha256: 8ac1a338655ea2b885704f73328a2581f975154dfc8042de5bb7166fe2ef277e
actual-sha256: c41a179737d21fae61582b0f90d32238b592aecaf2116691151b6ffbb59d60c5
--- upstream.expected
+++ tolk.actual
@@ -153,7 +153,7 @@
       float val83 = (*(data24_256+(Lidx13+88)));
       float val84 = (*(data24_256+(Lidx13+96)));
       float val85 = (*(data24_256+Lidx13));
-      *(data0_64+((Lidx12<<3)+Lidx13)) = ((*(buf2+0))+(*(buf10+0))+(*(buf9+0))+(*(buf8+0))+(*(buf7+0))+(*(buf6+0))+(*(buf5+0))+(*(buf4+0))+(*(buf3+0))+(*(buf1+0))+(*(buf0+0))+(val31*val85)+(val27*val81)+(val7*val61)+(val18*val72)+(val21*val75)+(val22*val76)+(val23*val77)+(val24*val78)+(val25*val79)+(val26*val80)+(val28*val82)+(val29*val83)+(val30*val84)+(val0*val54)+(val1*val55)+(val2*val56)+(val3*val57)+(val4*val58)+(val5*val59)+(val6*val60)+(val8*val62)+(val9*val63)+(val10*val64)+(val11*val65)+(val12*val66)+(val13*val67)+(val14*val68)+(val15*val69)+(val16*val70)+(val17*val71)+(val19*val73)+(val20*val74));
+      *(data0_64+((Lidx12<<3)+Lidx13)) = ((*(buf2+0))+(*(buf10+0))+(*(buf9+0))+(*(buf8+0))+(*(buf7+0))+(*(buf6+0))+(*(buf5+0))+(*(buf4+0))+(*(buf3+0))+(*(buf1+0))+(*(buf0+0))+(0.0f+(val31*val85)+(val27*val81)+(val7*val61)+(val18*val72)+(val21*val75)+(val22*val76)+(val23*val77)+(val24*val78)+(val25*val79)+(val26*val80)+(val28*val82)+(val29*val83)+(val30*val84)+(val0*val54)+(val1*val55)+(val2*val56)+(val3*val57)+(val4*val58)+(val5*val59)+(val6*val60)+(val8*val62)+(val9*val63)+(val10*val64)+(val11*val65)+(val12*val66)+(val13*val67)+(val14*val68)+(val15*val69)+(val16*val70)+(val17*val71)+(val19*val73)+(val20*val74)));
     }
   }
 }
